commands/ci/debug.js

```diff
@@ -76,13 +75,7 @@ function* run (context, heroku) {
     showStatus: false
   })
 
-  let dynoPromise
-  if (attachURL) {
-    dyno.dyno = { attach_url: attachURL }
-    dynoPromise = dyno.attach()
-  } else {
-    dynoPromise = dyno.start()
-  }
+  dyno.dyno = { attach_url: Utils.dig(testNodes, 0, 'dyno', 'attach_url') }
```

Comment:
What happens if no test nodes are returned? We have plans to wait on creating the test nodes until the App Setup has finished. I know it requires changes in L67, so maybe it's a discussion for another time.